source/module_base/libm/libm.h

```diff
@@ -0,0 +1,67 @@
+//==========================================================
+// AUTHOR : alcanderian@gmail.com
+// DATE : 2023-01-06
+//==========================================================
+
+#ifndef MODULE_BASE_ABAUCS_LIBM_H
+#define MODULE_BASE_ABAUCS_LIBM_H
+
+#include <complex>
+#include <cmath>
+
+#if !defined(__INTEL_COMPILER) && defined(USE_ABACUS_LIBM)
+#define WITH_ABACUS_LIBM
+#pragma message("we are using self libm now!")
```

Comment:
Here is a compile time message to comfirm if our libm source is built.
Is it acceptable?

Reply:
> Here is a compile time message to comfirm if our libm source is built. Is it acceptable?

I think it would be better done in CMakelists, i.e. using CMake to detect whether Intel compiler is used, and give out a prompt on it.

Reply:
Ok, I will try